trojsten/rules/kms.py

```diff
@@ -1,15 +1,20 @@
 # -*- coding: utf-8 -*-
 
 import datetime
+from collections import namedtuple
+from logging import getLogger
 
 from django.db.models import Count, Q
 from django.utils import timezone
 
+from trojsten.contests.models import Semester
 from trojsten.events.models import EventParticipant
 from trojsten.people.constants import SCHOOL_YEAR_END_MONTH
 from trojsten.results.constants import COEFFICIENT_COLUMN_KEY
 from trojsten.results.generator import CategoryTagKeyGeneratorMixin, ResultsGenerator
+from trojsten.results.manager import get_results, get_results_tags_for_rounds
 from trojsten.results.representation import ResultsCell, ResultsCol, ResultsTag
+from trojsten.rules.ksp_levels import get_total_score_column_index
```

Comment:
Čo si myslíš o tom, že túto funkciu by sme presunuli z ksp_levels do samostatného helpers.py súboru a importovali v ksp_levels aj tu odtiaľ? Myslím že by to mohlo predísť problémom ak sa to náhodou rozhodnú v ksp meniť.

Reply:
Určite môžeš presunúť, je to všeobecnejšia funkcia než len pre KSP. Možno je to lepšie dať do `results/helpers.py` ako do `rules/helpers.py`, ale to už nechám na tebe.

Reply:
Pravda, to dava viac zmyslu